contracts/test/ETO/MockETOCommitment.sol

```diff
@@ -72,4 +72,14 @@ contract MockETOCommitment is
         emit LogTermsSet(msg.sender, address(etoTerms), address(equityToken));
         emit LogETOStartDateSet(msg.sender, startAt, logStartDate);
     }
+
+    //
+    // Override IAgreement internal interface to allow mocking up agreements for fixtures
+    //
+    function mCanAmend(address /*legalRepresentative*/)
```

Comment:
why is this in here now? It is not related to the decimals issue, is is?

Reply:
yes, my fault but let me keep it - we'll use it on the backend fixtures generator to amend all agreements to autogenerated - important for testing